charts/hub/ci/ct-values.yaml

```diff
@@ -0,0 +1 @@
+fullnameOverride: hub
```

Comment:
Needs newline at EOF

Reply:
Ah but also, I just said `hub` as an example, I see the chart name is `artifact-hub`. I personally think we should keep the chart directory name the same as the chart name for clarity to end users.

At risk of bikeshedding here, this brings up a question to me: should the chart name be `artifact-hub` or `artifacthub` to be consistent with the github org name? This is probably a question for @mattfarina, @tegioz and @cynthia-sg as they maintain this project.

Reply:
We can rename it to `artifacthub`, sounds good to me 👍

Reply:
to condense replies: All above issues are resolved (i believe). I also fixed the secret to match the newer template of host for postgresql. I changed the name of the chart to artifacthub as decided, and fixed the commit issue. I leave the `Resolve conversation` buttons to the creators so i dont step on any toes. Lemme know if you see anything else!.